Gopkg.toml

```diff
@@ -39,7 +39,7 @@
 
 [[constraint]]
   name = "github.com/giantswarm/opsctl"
-  version = "9923241.0.0"
+  branch = "master"
```

Comment:
Do we have opsctl dependency here? This doesn't look good.

Reply:
yes, OpsGenie service only(no other packages are vendored).

Reply:
This looks like we should extract some common opsgenie library. Or at least move this part to `pkg` in `opsctl` to indicate this is used outside.